src/routes/contactadmin.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const { getAdmins, getAdminById, editAdmin } = require("../controllers/contactadmin");
+const container = require("../middlewares/container")
+ 
+const router = express.Router();
+ 
+router.get('/All', container(getAdmins));
+router.get('/:messageID', container(getAdminById));
```

Comment:
Param name needs to be in camelCase
Same below

Reply:
Fixed